CombiningObjects/my-contact-activities/my-contact-activities/definition/objectDefinition.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
@kiawnna is the duplicate directory names (`/my-contact-activities/my-contact-activities`) a nuance with the doctor?

Reply:
@nick-bair Updated all of the Readmes with correct commands and removed duplicate structure.